generate_protos.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Would it make sense to have the protos in subdirs that match their packages? Then it should be sufficient to just specify the top-level directory as proto_path, no? WDYT

Generally, we can have services that depend on other services and use its protos. So, dependency can get arbitrarily complex..

Reply:
Yes yes yes, Absolutely. And for our core repository too.
Our current, protos tree is not aligned with `protoc` package resolution.

It was already messy when I introduced a proto-linter. And seems, it becomes even worse now.
I don't remember if I created a task for that. But I believe that we must follow `protoc` guidances.

I will use this repository as an example.

Reply:
Please, check [changes](https://github.com/gopollock/public-api/pull/1/commits/fc93a88b0befc988b8eb503be5a6228b35758fa8). I hope it makes sense for you and we will apply it to classtime mono-repo too.

Generally, we are not required to keep the package name location in sync. It's just important that (`--proto_path` + import path in .proto files) == real file location.

Reply:
We might need to spend some time on the Kotlin generation part and the IntelliJ config part. So I have created a task: https://app.asana.com/0/1177833412654353/1205113245208557